core-blocks/code/edit.native.js

```diff
@@ -0,0 +1,28 @@
+import { View } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { PlainText } from '../../blocks';
+
+export function edit( { attributes, setAttributes, style } ) {
+	return (
+		<View>
```

Comment:
Shouldn't be `style` applied to `View` rather than `PlainText` to mirror web implementation?

Reply:
Yes and no. Unfortunately, styling in RN is not cascading by default and we will need more work on establishing a pattern on how to pass and style the blocks/components/containers.

In this particular case for example, the style includes the `fontFamily` property which is supported by the `PlaintText` component (because it's a `TextInput` under the hood) but by the `View` component.

All in all, styling should be considered `alpha` at this stage as well and i'd suggest we work on it in a different PR, if that's OK with you too @gziolo . I'll update the PR description to include some relative info.

Reply:
That would be awesome to have it documented in the code next to RN version of the `edit` component.

Reply:
Done with 0377c73.